pages/docs/database-connections/role-based-credentials.mdx

```diff
@@ -0,0 +1,70 @@
+---
+description: Role-based data credentials allow you to control what users can access from the same data connection.
+---
+
+import { Callout } from "nextra-theme-docs";
+
+# Role-Based Database Credentials
+
+All Hashboard data connections can be optionally configured to use additional credentials which can be assigned to users on a per-role basis. 
+
+After setting up database-level permissions within the data warehouse you use with Hashboard, these credentials can be added within Hashboard and then assigned to your Hashboard project's roles.
+
```

Comment:
I think we can do without this sentence? Seems essentially equivalent to what's above.